be/src/connector/hive_connector.cpp

```diff
@@ -47,6 +47,16 @@ DataSourcePtr HiveDataSourceProvider::create_data_source(const TScanRange& scan_
 HiveDataSource::HiveDataSource(const HiveDataSourceProvider* provider, const TScanRange& scan_range)
         : _provider(provider), _scan_range(scan_range.hdfs_scan_range) {}
 
+Status HiveDataSource::_check_all_slots_nullable() {
```

Comment:
Will it cause some of the previous cases to report errors? Our some test cases have non-nullable columns in iceberg array types.

Reply:
Looks like not. be ut has passed. 

but after seocnd thought, we'd better to support non-nullable column.